extra/startup.sh

```diff
@@ -130,6 +151,18 @@ _() {
     pip install -U cryptography
     pip install ansible==2.7
 
+
+    # Get WORKER_KEY from Vault
+    if [[ -z "${WORKER_KEY}" ]] && [[ -n "${VAULT_SECRET_ID}" && -n "${VAULT_ROLE_ID}" ]] ; then
+        TOKEN=$(curl -sSL --request POST --data "{\"role_id\":\"$VAULT_ROLE_ID\",\"secret_id\":\"$VAULT_SECRET_ID\"}" $VAULT_URL/v1/auth/approle/login | jq -r '.auth.client_token')
+        export WORKER_KEY=$(curl -sSL -H "X-Vault-Token: $TOKEN" -X GET $VAULT_URL/v1/cycloid/$TEAM_ID/cycloid-worker-keys | jq  -r .data.ssh_prv | base64 -w0)
+    fi
+
+    # Make sure WORKER_KEY looks ok
+    if ! echo $WORKER_KEY | base64 -d | grep "PRIVATE KEY"; then
```

Comment:
instead of grep, maybe we could validate the key with `ssh-keygen` or `openssl`.

Reply:
Can you check last commit ?